### ISSUE TITLE
Add superseeding link to rdamsc.bath.ac.uk  ?

### ISSUE THREAD

**Issue by stain** [rd-alliance/metadata-directory#81]
The [GitHub README](https://github.com/rd-alliance/metadata-directory/blob/gh-pages/README.md) links to v2 at https://rdamsc.bath.ac.uk/ - but there is no link to that from https://rd-alliance.github.io/metadata-directory/ web content.  So only someone trying to contribute might found out about v2.

Therefore new efforts like [RDMKit](https://rdmkit.elixir-europe.org/metadata_management.html) continue to link to this instance of the directory (which perhaps is more edible, given that https://rdamsc.bath.ac.uk/scheme-index does not render any descriptions, unlike the nicer https://rd-alliance.github.io/metadata-directory/standards/)

I'm fine with this one being normative again - but then <https://rdamsc.bath.ac.uk/> should point back here.

**Comment by alex-ball** [comment]
Thanks for this.

As a short term measure, MSD and MSC are being run in parallel. Once MSC v2 is complete, there will be an audit of the content of all three incarnations of this resource; they will be brought into sync and the MSD will be officially deprecated, and this will be made obvious from the web content. MSC is emphatically where all future development will take place.

Regarding the directory page you link to, the true equivalents on MSC are actually:

- https://rdamsc.bath.ac.uk/subject/Arts%20and%20humanities
- https://rdamsc.bath.ac.uk/subject/Engineering
- https://rdamsc.bath.ac.uk/subject/Biological%20sciences
- https://rdamsc.bath.ac.uk/subject/Physical%20sciences
- https://rdamsc.bath.ac.uk/subject/Mathematics%20and%20statistics
- https://rdamsc.bath.ac.uk/subject/Social%20and%20human%20sciences
- https://rdamsc.bath.ac.uk/subject/Multidisciplinary

You are welcome to raise any issues with MSC over on its own issue tracker.
